lib/SPIRV/SPIRVLowerMemmove.cpp

```diff
@@ -46,17 +46,17 @@
 #include "llvm/IR/Module.h"
 #include "llvm/IR/PassManager.h"
 #include "llvm/Pass.h"
+#include "llvm/Transforms/Utils/LowerMemIntrinsics.h"
 
 using namespace llvm;
 using namespace SPIRV;
 
 namespace SPIRV {
 
-class SPIRVLowerMemmoveBase : public InstVisitor<SPIRVLowerMemmoveBase> {
+class SPIRVLowerMemmoveBase {
```

Comment:
Can we drop the `InstVisitor.h` include now?

Reply:
Sure.